examples/storm-hdfs-examples/src/main/java/org/apache/storm/hdfs/bolt/SequenceFileTopology.java

```diff
@@ -14,6 +14,8 @@
 
 import java.io.FileInputStream;
```

Comment:
Remove unused imports?

Reply:
Yes, will get this done. I'll probably use my IDE to organize the imports, so it might bloat the diff. Will put it in a new commit.

Reply:
Should be fixed now, fixed imports for all modules. The changes are in the second commit.